runtime/runtime/src/state_viewer.rs

```diff
@@ -214,6 +222,7 @@ mod tests {
             &mut logs,
             &MockEpochInfoProvider::default(),
             PROTOCOL_VERSION,
+            0x99,
```

Comment:
Please reuse a constant

Reply:
The part that I don't understand is if runtime should be depending on chain/chain-configs where genesis is... given genesis is pretty much runtime configuration as much as chain configuration....